draft-ietf-rats-corim.md

```diff
@@ -1128,21 +1123,38 @@ the object relates to the subject.
 #### Device Identity Triple {#sec-comid-triple-identity}
 
 A Device Identity triple relates one or more cryptographic keys to a device.
-The subject of an Identity triple uses an instance or class identifier to refer
-to a device, and a cryptographic key is the object. The predicate asserts that
-the identity is authenticated by the key. A common application for this triple
-is device identity.
+The triple subject is an identity used to name a device.
+The triple object is an array of cryptographic key(s) used to authenticate the device.
+The predicate asserts that any of the key(s) (found in the triple object) can be used to authenticate the device.
+
+If contained in a CoRIM, this triple endorses that the key(s) were provisioned to the Target Environment.
+
+If contained in Evidence, this triple asserts that the key(s) are present in the Target Environment.
+
+The Verifier SHOULD attempt to verify the status and validity of the key(s).
+
+Depending on which `$crypto-key-type-choice` form is supplied, the Verifier may
+need to take different steps to locate or verify the key.
 
 ~~~ cddl
 {::include cddl/identity-triple-record.cddl}
 ~~~
 
 #### Attestation Keys Triple {#sec-comid-triple-attest-key}
 
-An Attestation Keys triple relates one or more cryptographic keys to an
-Attesting Environment. The Attestation Key triple subject is an Attesting
-Environment whose object is a cryptographic key. The predicate asserts that the
-Attesting Environment signs Evidence that can be verified using the key.
+An Attestation Keys triple relates one or more cryptographic keys to an Attesting Environment.
+The attest key triple subject identifies an Attesting Environment
+whose object is an array of cryptographic key(s) used to attest Evidence.
```

Comment:
attest the Evidence on line 1147 is confusing and not really ties to any Cryptographic operation.